internal/resources/resource_host.go

```diff
@@ -590,6 +600,11 @@ func resourceMetalHostRead(d *schema.ResourceData, meta interface{}) (err error)
 	d.Set(hCHAPUser, host.ISCSIConfig.CHAPUser)
 	d.Set(hCHAPSecret, host.ISCSIConfig.CHAPSecret)
 	d.Set(hInitiatorName, host.ISCSIConfig.InitiatorName)
+
+	if len(host.WWPNs) > 0 {
+		_ = d.Set(hWWPNS, host.WWPNs)
```

Comment:
```suggestion
		if err := d.Set(hWWPNS, host.WWPNs); err != nil {
		    return fmt.Errorf("set WWPNs: %v", err) 
		}
```

Reply:
Hi @ron-saito, Please see this comment from Mike - https://github.com/HewlettPackard/hpegl-metal-terraform-resources/pull/176/files/b7e0b747020293dd264e681bea394e6506632e6d#r1520373211

It can be possible that iSCSI config or WWPNs can be empty. I added this check as per the suggestion given by Mike.

Reply:
@ron-saito please file defect if your comments are addressed here.

Reply:
I think all the d.Set() calls with the returns ignored were written before the lint check enabled on PRs.  I think all d.Set() calls should be checked.  if this Set call fails, it would likely cause terraform to think something changed.
While I agree a separate PR could address all d.Set calls, new code added should check the return.

Reply:
That is fine. @chitranm please make the changes as Ron suggested. Thanks.

Reply:
Sure @mchuang3.